setup.cfg

```diff
@@ -42,7 +42,7 @@ install_requires =
     appdirs>=1.4.4,<2
     configupdater>=1.1.3,<2
     setuptools>=38.3
-    setuptools_scm>=4.1.2,<5
+    setuptools_scm>=5,<6
```

Comment:
I was stumbling about this failing test already. Just didn't have the time to submit the bug report.
No big deal but good to know that it is fixed :smiley: